vivarium/core/process.py

```diff
@@ -384,6 +385,7 @@ def generate_topology(self, config: Optional[dict] = None) -> Topology:
 
     def generate_flow(self, config: Optional[dict] = None) -> Flow:
         _ = config
+        _ = self # Silence pylint no-self-use
```

Comment:
Does this mean this should be a static method instead? Seems like an indicator that it's warning us we're not using `self`.

Reply:
`self` is useful because users might want to pull attributes from `self` in their own implementations of these methods. On the flip side, I refactored three methods in `engine.py` to be standalone functions because they should never be overridden by end users and do not reference `self`.